html.go

```diff
@@ -123,6 +130,10 @@ func HtmlRendererWithParameters(flags int, title string,
 		renderParameters.FootnoteReturnLinkContents = `<sup>[return]</sup>`
 	}
 
+	if renderParameters.SanitizedAnchorNameOverride == nil {
+		renderParameters.SanitizedAnchorNameOverride = markdownutils.CreateGitLabAnchor
```

Comment:
What's the motivation to change the default? This seems unnecessary. Users will be able to set it to `markdownutils.CreateGitLabAnchor` or another custom func by using the `SanitizedAnchorNameOverride` parameter.